docs/ops/arithmetic/Sin_1.md

```diff
@@ -22,11 +34,6 @@
 
```

Comment:
Please add information about supported input tensor ranks. Are they of arbitrary size or there are some limitations?